CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+Version 1.16.1
+==============
+**June 2, 2023**
+
+This release contains a small number of enhancements and important bug fixes. Many thanks to @philippjfr
```

Comment:
I would swap the order round, first mention the important bug fixes then the small enhancements...